ddtrace/internal/writer/writer.py

```diff
@@ -281,14 +281,15 @@ def _reset_connection(self):
                 self._conn.close()
                 self._conn = None
 
-    def _put(self, data, headers, client):
-        # type: (bytes, Dict[str, str], WriterClientBase) -> Response
+    def _put(self, data, headers, client, no_trace=True):
```

Comment:
nit: Can we make this a required argument. Optional arguments can make this behavior easier to miss.